doc/source/examples/extended_examples/beam_analysis_with_reporting/beam_analysis_report.rst

```diff
@@ -0,0 +1,571 @@
+    :language: python
+    :dedent: 4
+    :start-at: # Apply distributed load as nodal forces
+    :end-at: mapdl.f(last_node, "FY", end_nodal_force)
+
+This approach:
+
+* Converts the continuous load to discrete nodal forces
+* Maintains load equilibrium
+* Provides accurate representation of the distributed loading
+
+Alternatively, the load can be applied using the ``SFBEAM`` command for surface loads on BEAM and PIPE elements.
+
+Solution Process
+================
+
+The static structural solution is configured and executed:
+
+.. literalinclude:: ../../../../../examples/00-mapdl-examples/beam_with_report.py
+    :language: python
+    :dedent: 4
+    :start-at: # Enter solution processor
+    :end-at: print("Solution completed successfully")
+
+The solution uses:
+
+* Static analysis type (no time-dependent effects)
+* Linear analysis (small deflection theory)
+* Direct solver for optimal accuracy
+
+Post-Processing and Result Extraction
+=====================================
+
+Displacement Results
+--------------------
+
+The maximum displacement is extracted using the `post_processing` module:
+
+.. literalinclude:: ../../../../../examples/00-mapdl-examples/beam_with_report.py
+    :language: python
+    :dedent: 4
+    :start-at: # Get nodal displacements
+    :end-at: max_displacement_location = np.argmin(displacements) + 1
+
+Moment Results
+--------------
+
+The Moment at both nodes are extracted using ``ETABLE`` command, with ``SMISC`` items:
+
+.. literalinclude:: ../../../../../examples/00-mapdl-examples/beam_with_report.py
+    :language: python
+    :dedent: 4
+    :start-at: # Get moment values
+    :end-at: max_moment_fem = max(max_moment_i, max_moment_j)
+
+Bending Stress Results
+----------------------
+
+Again ``ETABLE`` is used to retrieve MAPDL values:
+
+.. literalinclude:: ../../../../../examples/00-mapdl-examples/beam_with_report.py
+    :language: python
+    :dedent: 4
+    :start-at: # Get bending stress values
+    :end-at: max_stress_fem = max(bending_stress_top, bending_stress_bottom)
+
+Bending Strain Results
+----------------------
+
+.. literalinclude:: ../../../../../examples/00-mapdl-examples/beam_with_report.py
+    :language: python
+    :dedent: 4
+    :start-at: # Get bending strain values
+    :end-at: max_strain_fem = max(bending_strain_top, bending_strain_bottom)
+
+Report Generation Features
+==========================
+
+Markdown Report
+---------------
+
+The Markdown report includes:
+
+* Executive summary with key results
+* Detailed model description with parameter tables
+* Comprehensive results presentation
+* Analytical verification section
+* Professional formatting with tables and equations
+
+.. literalinclude:: ../../../../../examples/00-mapdl-examples/beam_with_report.py
+    :language: python
+    :start-at: def generate_markdown_report(data, output_dir):
+    :end-at: return str(report_file)
+
+Word Document Report
+--------------------
+
+The Word report provides:
```

Comment:
```suggestion
The Word document report provides:
```
Either remove "document" from title or add it in the sentence so the terminology is the same.